README.md

```diff
@@ -26,13 +26,15 @@ If you are new to React Native, this is a helpful introduction: https://facebook
 
 4. Run `yarn nodeify`.
```

Comment:
i think it's no longer needed. 

Also, the part about `nvm` should be optional. Personally, i don't use it.

Reply:
Then we need to specify the node version, there are certain things like Ganache that doesn't work with node 14.
Node 10 & 12 works fine.